modules/EasyAbp.EShop.Stores/src/EasyAbp.EShop.Stores.Web/Pages/EShop/Stores/Stores/Store/EditModal.cshtml.cs

```diff
@@ -16,17 +23,36 @@ public class EditModalModel : StoresPageModel
         [BindProperty]
         public CreateEditStoreViewModel Store { get; set; }
 
+        public ICollection<SelectListItem> StoreOwners { get; set; }
+
         private readonly IStoreAppService _service;
+        private readonly IIdentityUserAppService _userAppService;
+        private readonly IStoreOwnerAppService _storeOwnerAppService;
 
-        public EditModalModel(IStoreAppService service)
+        public EditModalModel(IStoreAppService service,
+            IIdentityUserAppService userAppService,
+            IStoreOwnerAppService storeOwnerAppService)
         {
             _service = service;
+            _userAppService = userAppService;
+            _storeOwnerAppService = storeOwnerAppService;
         }
 
         public async Task OnGetAsync()
         {
+            StoreOwners =
+                (await _userAppService.GetListAsync(new GetIdentityUsersInput
+                { MaxResultCount = LimitedResultRequestDto.MaxMaxResultCount })).Items
+                .Select(x => new SelectListItem(x.UserName, x.Id.ToString())).ToList();
+
```

Comment:
What if there are more than 1000 users?

Reply:
I just follow the implementation of product categories, see this:
https://github.com/EasyAbp/EShop/blob/ac750c4ee65d32dbe3693eea339ce8645bf3eb38/modules/EasyAbp.EShop.Products/src/EasyAbp.EShop.Products.Web/Pages/EShop/Products/Products/Product/EditModal.cshtml.cs#L56-L64

Reply:
The count of ProductType and Category is expected to be less than 1000, but the count of User is easy to be more than 1000.

Reply:
I think the best practice should be selecting from a modal with filtered user list, but I have no clue how to implement filtered list in abp, do you have any examples?

Reply:
Maybe using UserId directly before we have a better way.

Reply:
Agreed.